Sources/Swift/Integrations/SessionReplay/SentryBreadcrumbReplayConverter.swift

```diff
@@ -0,0 +1,92 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryBreadcrumbReplayConverter: NSObject {
```

Comment:
Probably nitpicking, but: should we name it `SentryReplayBreadcrumbConverter` rather? Because now it's sounds like we're converting breadcrumb replays and not replay breadcrumbs 😅